streaming/java/streaming-state/src/main/java/org/ray/streaming/state/config/ConfigHelper.java

```diff
@@ -0,0 +1,74 @@
+package org.ray.streaming.state.config;
+
+import java.util.Map;
+
+/**
+ * Config Helper figure out the config info.
+ */
+public class ConfigHelper {
```

Comment:
Actually I consider if we can reuse some existing config library like typesafe's Config or we properly design a general Config module instead of Map + ConfigHelper inside ray/streaming/java such an inner scope.

Reply:
Yes, I agree with you. The config module is a better way. But the whole configuration obtain method now is not designed. I use some helper class temporary.

Reply:
Make sense. But in this we shall add as least method as needed. I can see a lot of methods like

```
public static long getLong(Map config, String configKey);
public static long getLongOrDefault(Map config, String configKey, long defaultValue);
public static boolean getBoolean(Map config, String configKey);
```

are never into use. We said that it is a temporary hack and I believe an issue will be opened which lead to introduce a config library and refactor these codes in the near future.

Reply:
FYI https://github.com/ray-project/ray/issues/7485

Reply:
yes, i'll clean this codes

Reply:
> Yes, I agree with you. The config module is a better way. But the whole configuration obtain method now is not designed. I use some helper class temporary.

As far as global project configuration, maybe you can talk to @BalaBalaYi who proposed owner-config in streaming runtime already.